build.xml

```diff
@@ -2250,11 +2337,27 @@
                   reportLevel="low"
                   >
             <sourcePath path="${source}/"/>
-            <class location="${jartarget}/jmri.jar"/>
+            <class location="${jartarget}/jmri-tests.jar"/>
```

Comment:
Just checking. Should target `spotbugs-ci` use `jmri-tests.jar` and not `jmri.jar`?

Reply:
Thanks for catching that.  Fixed.